demos/2019_RHTE/files/cpma.yaml

```diff
@@ -0,0 +1,56 @@
+---
+- hosts: localhost
+  connection: local
+  tasks:
+  - name: "Reading v4 variables"
+    include_vars: "../4.x/my_vars.yml"
+
+  - name: "Setting facts"
+    set_fact:
+      guid_v4: "{{ guid }}"
+      subdomain_v4: "{{ subdomain_base_suffix }}"
+
+  - name: "Reading v3 variables"
+    include_vars: "../3.x/my_vars.yml"
+
+  - name: "Setting facts"
+    set_fact:
+      guid_v3: "{{ guid }}"
+      subdomain_v3: "{{ subdomain_base_suffix }}"
+      output_dir_v3: "{{ output_dir }}"
+
+  - name: "Registering host"
+    add_host:
+      hostname: "bastion.{{ guid_v3 }}{{ subdomain_v3 }}"
+      groups: "remote"
+
+- hosts: remote
+  vars_files:
+    - "../3.x/my_vars.yml"
+  vars:
+    ansible_ssh_private_key_file: "{{ output_dir }}/{{ guid }}key"
+    ansible_user: ec2-user
+  tasks:
+  - name: "Including v4 variables"
+    include_vars: "../4.x/my_vars.yml"
+    delegate_to: localhost
+
+  - name: Copy ssh key to bastion host
+    copy:
+      src: ~/.ssh/libra.pem
```

Comment:
I think we'll want to revisit the ssh key so this can work for the labs.
Expect labs will use their own ssh key.

Assume we can grab the key name from my_vars.yml
https://github.com/fusor/mig-agnosticd/blob/master/3.x/my_vars.yml.sample#L20